### DISCUSSION TITLE
What about a frictionless badge?

### DISCUSSION THREAD

**Discussion by aborruso** [frictionlessdata/frictionlessdata.io#677] [Ideas] [answered]
Hi,
it probably already exists. If yes where is it?

If no, I think it would be great to have it in frictionless website. A simple note "if you publish frictionless data, you can tell the world by using this badge"

[![](https://img.shields.io/badge/frictionless-ready-%23EA6D4C)](https://frictionlessdata.io/)

```
[![](https://img.shields.io/badge/frictionless-ready-%23EA6D4C)](https://frictionlessdata.io/)
```

**Comment by roll** [comment, marked as answer]
@aborruso 
It's a great idea!

And actually, we're thinking about a project which will automatically validate data on Github providing those badges. I hope it's about to be prototyped and announced very soon.

**Reply by roll** [reply]
I'll mark my answer as accepted by basically @aborruso answered it in the question also as the Shields.io is way to go ATM

**Reply by aborruso** [reply]
@roll what about to add a note about it, somewhere in one of the frictionless website

**Reply by roll** [reply]
@aborruso 
Good idea cc @lwinfree

**Reply by loleg** [reply]
For instructions, see https://repository.frictionlessdata.io/docs/badges.html
